doc/docs/master/enterprise/index.md

```diff
@@ -1,11 +1,15 @@
 # Enterprise Edition
 
 This section describes features available in the enterprise
-edition of Pachyderm. You can try them by enabling a
-14-day free-trial by logging in to the Pachyderm UI and
-providing your email address. If you are interested in
-purchasing Pachyderm Enterprise, contact our sales team.
+edition of Pachyderm. 
+
+!!! Warning "Get your free-trial token."
+    To try our enterprise features, fill in [this form](https://www.pachyderm.com/trial) or log in to the Pachyderm UI and
+    provide your email address. 
 
+
+If you are interested in
+purchasing Pachyderm Enterprise, contact our sales team.
```

Comment:
`If you're interested in Pachyderm Enterprise, [schedule some time](https://www.pachyderm.com/request-a-demo/) with one of our experts.